eng/pipelines/templates/jobs/ci.yml

```
@@ -140,15 +131,44 @@ jobs:
       image: $(LINUXVMIMAGE)
       os: linux
 
-    variables:
-      ComponentDetection.SourcePath: '$(Build.SourcesDirectory)/sdk/${{ parameters.ServiceDirectory }}'
+    steps:
+    - template: /eng/pipelines/templates/steps/download-package-artifacts.yml
+
+    - template: /eng/pipelines/templates/steps/resolve-package-targeting.yml
+      parameters:
+        BuildTargetingString: ${{ parameters.BuildTargetingString }}
+        PackagePropertiesFolder: $(Build.ArtifactStagingDirectory)/PackageInfo
+
+    - template: ../steps/build-extended-artifacts.yml
```

```suggestion
    - template: /eng/pipelines/templates/steps/build-extended-artifacts.yml
```
nit and can be done in follow-up.